tests/moduleTests/pitch.js

```diff
@@ -64,4 +65,205 @@ export default function tests() {
 
     });
 
+    test('music21.pitch.To String', assert => {
+        const p = new music21.pitch.Pitch('B#3');
+        assert.equal(p.toString(), '<Pitch B#3>', 'Equal');
+    });
+    
+    test('music21.pitch.Pitch Equality', assert => {
+        const pitch_pairs = [
+            ['a#', 'a', false],
+            ['a#', 'b-', false], 
+            ['a#', 'a-', false], 
+            ['a##', 'a#', false],
+            ['a#4', 'a#4', true], 
+            ['a-3', 'a-4', false], 
+            ['a#3', 'a#4', false]
+        ];
+        
+        for (const pair of pitch_pairs) {
+            const p1 = new music21.pitch.Pitch(pair[0]);
+            const p2 = new music21.pitch.Pitch(pair[1]);
+            if (!pair[2]) {
```

Comment:
great work so far! this is just an if/else, so no need for the `else if` -- you want this structure:
```
if (pair[2]) {
    assert.deepEqual(p1, p2, 'Equal');
} else {
    assert.notDeepEqual(p1, p2, 'Not Equal');
}
```
(what you've already written is technically correct, but unclear.)

please also note that the style throughout this project is to put the `else` on the same line as the closing bracket of the `if` block

Reply:
That's my mistake, it should be all fixed now.

Reply:
Thanks!

Now this is _super_ nitpicky, but... there's no reason to negate the value you're checking on and put the "not equal" branch first. It's easier to read the way I've laid it out above (with the 'true' branch first).

Super trivial in this case, but in more complex cases this can make your code easier for a reader to understand.

Reply:
That makes total sense, I think just copied the order from the python test case when I was modifying to fit JS logic. I'll adjust the order now.